spec/javascripts/govuk-component/option-select-spec.js

```diff
@@ -86,6 +86,10 @@ describe('GOVUK.OptionSelect', function() {
     expect(optionSelect.isClosed()).toBe(false);
   });
 
+  it ('has called setupHeight as part of initialisation', function(){
```

Comment:
Can we make the test name to what we're testing, which is that it's expanded by default, and update the test to reflect this?